packages/atomic/src/components/atomic-pager/atomic-pager.tsx

```diff
@@ -112,6 +119,10 @@ export class AtomicPager implements InitializableComponent {
 
   private buildPage(page: number) {
     const isSelected = this.pager.isCurrentPage(page);
+    const parts = ['page-button'];
+    if (isSelected) {
+      parts.push('active-page-button');
+    }
```

Comment:
`false` was being added as a part

Reply:
To be tested, but if you have 2 parts, I think selecting a single one won't work with a CSS selector with cypress test, is that true?

Reply:
If you do `atomic-pager[part*="active-page-button"]` it should be able to select it. 

Or do you mean something different @ThibodeauJF  ?

Reply:
Yeah, that's what I assumed. I just tested and we aren't able to query the element when there's two parts. One possible fix would be to introduce a `.part(...)` util for Cypress, but I don't think that's necessary yet.